src/Resources/config/app/config.yml

```diff
@@ -2,6 +2,11 @@ parameters:
     sylius_refund.refund_enum_type: Sylius\RefundPlugin\Entity\Type\RefundEnumType
     sylius_refund.refund_type: Sylius\RefundPlugin\Model\RefundType
 
+sylius_refund:
+    pdf_generator:
+        allowed_files:
```

Comment:
Why can't you just use `knp_snappy.pdf.options.allow` and allow there for the logo instead of defining the new parameter?

Reply:
I was thinking about it and did it that way because `knp_snappy.pdf.options.allow` applies to all pdf generators that use KnpSnappy. If someone already uses KnpSnappy in their project, may want to have different allowed files for their own generator, and different for the generator coming from Sylius 🤔
But I don't know my considerations make any sense so I'm open to applying your suggestion.

Reply:
Oh, I think I've even tried it that way. I could pass only strings to `knp_snappy.pdf.options.allow` so the env variable didn't work. But I'm not sure about it right now. I'll try again and confirm it.

Reply:
Looking at the configuration suggested in [our docs](https://docs.sylius.com/en/latest/cookbook/payments/customizing-credit-memo.html#how-to-change-the-logo-in-the-credit-memo), passing env variable should work 🤔  
```
knp_snappy:
    pdf:
        options:
            allow: '%env(resolve:SYLIUS_REFUND_LOGO_FILE)%'
```

Reply:
Yep, I've got an error:
> Invalid` type for path "knp_snappy.pdf.options.allow". Expected "scalar", but got "array".
For the following configuration:
```yaml
knp_snappy:
    pdf:
        options:
            allow: '%env(default:default_logo_file:resolve:SYLIUS_REFUND_LOGO_FILE)%'
```
but
```yaml
knp_snappy:
    pdf:
        options:
            allow: '%env(SYLIUS_REFUND_LOGO_FILE)%'
```
already works, however `SYLIUS_REFUND_LOGO_FILE` may not be set.

Reply:
What about `allow: '%env(default:default_logo_file:SYLIUS_REFUND_LOGO_FILE)%'` 👀

Reply:
It's assigning a plugin specific variable `SYLIUS_REFUND_LOGO_FILE` to a project-wide configuration.
How will the user configure paths unrelated to the plugin? What about the invoicing plugin?
I prefer plugin-owned configuration.

Reply:
I thought that there is a possibility to define an array under the `allow` config, I didn't pay attention to: 
> Invalid` type for path "knp_snappy.pdf.options.allow". Expected "scalar", but got "array".

Now, the addition plugin configuration makes sense to me 👌🏻